vm/actor/src/builtin/miner_actor.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![allow(dead_code)]
+use address::Address;
+use cid::Cid;
+use encoding::de;
+use libp2p::PeerId;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+pub struct StorageMinerActor {}
+
+/// Container representing storage miner actor state
+pub struct StorageMinerActorState {
+    // TODO add proving_set, post_state
+    /// The height at which this miner was slashed at.
+    slashed_at: u64,
+    /// Sectors this miner has committed
+    sectors: Cid,
+    /// Contains static info about this miner
+    info: Cid,
+}
```

Comment:
What is the specific reason to protect these fields? For something like this it may make sense to just have the fields public (should any interaction be protected?)

Reply:
No particular reason just defaulted to protecting, will make pub and remove getter methods to retrieve fields